src/sagemaker_huggingface_inference_toolkit/optimum_utils.py

```diff
@@ -38,21 +38,25 @@ def get_input_shapes(model_dir):
     # try to get input shapes from config file
     try:
         config = AutoConfig.from_pretrained(model_dir)
-        if hasattr(config, "neuron_batch_size") and hasattr(config, "neuron_sequence_length"):
-            input_shapes["batch_size"] = config.neuron_batch_size
-            input_shapes["sequence_length"] = config.neuron_sequence_length
-            input_shapes_available = True
-            logger.info(
-                f"Input shapes found in config file. Using input shapes from config with batch size {input_shapes['batch_size']} and sequence length {input_shapes['sequence_length']}"
-            )
-            if os.environ.get("HF_OPTIMUM_BATCH_SIZE", None) is not None:
-                logger.warning(
-                    "HF_OPTIMUM_BATCH_SIZE environment variable is set. Environment variable will be ignored and input shapes from config file will be used."
-                )
-            if os.environ.get("HF_OPTIMUM_SEQUENCE_LENGTH", None) is not None:
-                logger.warning(
-                    "HF_OPTIMUM_SEQUENCE_LENGTH environment variable is set. Environment variable will be ignored and input shapes from config file will be used."
+        if hasattr(config, "neuron"):
```

Comment:
And what if the neuron section doesn't exist in the config? Shall we raise or warn here?

Reply:
if it doesn't exist the env vars will be checked. Its just that the priority is config > env > error